### ISSUE TITLE
Can't move cursor while holding space

### ISSUE THREAD

**Issue by AeroNotix** [borodust/boomstick-fig#1]
as title

**Comment by borodust** [comment]
Thanks for the report! I'm unsure what the behavior is though. Do you mean sytem mouse cursor doesn't move whatsoever when you move mouse around while holding spacebar? Does this happen outside of the :boomstick-fig window as well?

**Comment by AeroNotix** [comment]
Yes, when I hold space the cursor doesn't move whatsoever, inside or outside the :boomstick-fig window. 

sbcl 1.4.0, linux 4.14.15

**Comment by borodust** [comment]
Hm-m, that seems like some sort of system or desktop manager behavior. Can you reproduce this issue in any other application (text editor or anything)?

**Comment by AeroNotix** [comment]
I've never seen this exact behaviour before in other applications but I am using XMonad which often doesn't play nice with games so I will try with something else and report back.

**Comment by borodust** [comment]
Did you manage to reproduce the issue outside of `:boomstick-fig`?
